src/sass/bootstrap4/_carousel.scss

```diff
@@ -190,6 +190,7 @@
   bottom: 20px;
   left: (100% - $carousel-caption-width) / 2;
   z-index: 10;
+  max-width: 550px;
```

Comment:
Your solution of changing the `left` to `50%` is much better than my `max-width` suggestion, so much so that I would say you can delete this style. I tested it and this one doesn't seem to make much of a difference when paired with the `left: 50%`. And it's one less hardcoded value I guess

Reply:
My thinking was` left: 50%` would be used in tandem with the max-width, as on larger displays the box can still extend `550px`. But at that point it'll be splitting hairs I guess, so you can choose to keep/remove that line at your discretion.

Reply:
Ok the `left: 50%` by itself prevents the caption from overlapping with the 5 indicator buttons, so it no longer matters whether the box extends past 550px. I'll remove it.